out/notebooks/splunk.ts

```diff
@@ -267,72 +302,33 @@ function handleErrorPayloads(data: any, statusCode: number) {
     });
 }
 
-export function getSearchJobBySid(service, sid) {
-    return new Promise(function(resolve, reject) {
-        service.getJob(sid, function(err, data) {
-            if (err != null) {
-                reject(err);
-            } else {
-                resolve(data);
-            }
-        });
-    });
+export function getSearchJobBySid(service, sid): Promise<any> {
+    let request = service.getJob(sid);
```

Comment:
All of these refactors were necessary after upgrading the Javascript SDK to version 2+ see: https://github.com/splunk/splunk-sdk-javascript?tab=readme-ov-file#migrate-from-callbacksv1x-to-promiseasync-awaitv2x

Reply:
so the resolve/reject handling that I removed is just part of the Promise formation and that's done in the SDK now. For these functions the expectation is that error handling is happening by the caller, which you can see happening here for the SPL2 case for example: https://github.com/splunk/vscode-extension-splunk/blob/4bda67da4524ca24610d66da6a765fce5741742c/out/notebooks/spl2/controller.ts#L37-L59